pkg/ddevapp/global_dotddev_assets/commands/host/dbeaver

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Run DBeaver against current db
+## Usage: dbeaver
+## Example: "ddev dbeaver [db] [user]"
```

Comment:
This needs a little work, a list of path-to-file would be good. I suspect it works on Windows as well, but don't have an easy way to test. But if we just test this on things we can test with it will be OK.

```suggestion
## Example: "ddev dbeaver [db] [user]"
## OSTypes: darwin,linux
## HostBinaryExists: <path/to/file>
```

Reply:
I agree with `OSTypes: darwin,linux` - will change it.
Checked DBeaver in Windows WM and the path is different, it is located in APPDATA, and I don't know how to write the path.

`HostBinaryExists` - if I write the path for macOS only, the command will not work on Linux.

And there is no way I can tell all the path for Linux, because DBeaver can be installed from snap, flatpak or repo.

Reply:
Normal macOS location is `/Applications/DBeaver.app`

What you do is put a command-separated list of locations, like heidisql has, see https://github.com/ddev/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/host/heidisql#L8

And we can put all the linux locations we can think of in there.

Reply:
The idea is not to clutter the list of commands if they don't even have dbeaver installed.

Reply:
Ok, I'll try to collect all possible locations.

Reply:
Here is a download page https://dbeaver.io/download/

I check all possible locations for Linux and added them, including Flatpak and snap (these two available only for DBeaver Community Edition).

Maybe there is some extra path for macOS, if someone installs one of these:
- https://dbeaver.com/download/lite/
- https://dbeaver.com/download/enterprise/
- https://dbeaver.com/download/ultimate/